CTOR-1956 Plugin(network::cisco::meraki::cloudcontroller::restapi): check api path (#4732)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.md

> Keep your API key safe as it provides authentication to all of your organizations with the API enabled. 
> If your API key is shared, you can regenerate your API key at any time. This will revoke the existing API key.

> **Base URI Notice**
>

> > The default base URI value *api.meraki.com* used in `MERAKIAPIHOSTNAME` macro will work for most of the world.

Reviewer: ```suggestion
> The default base URI value *api.meraki.com* used in the `MERAKIAPIHOSTNAME` macro will work for most of the world.
```

> > However, for organizations hosted in the following country, you need to override this value and specify the respective base URI instead:

Reviewer: ```suggestion
> However, for organizations hosted in the following countries, you need to override this value and specify the corresponding base URI instead:
```

>
> - Canada: https://api.meraki.ca/api/v1
> - China: https://api.meraki.cn/api/v1
> - India: https://api.meraki.in/api/v1

> > - United States FedRAMP: https://api.gov-meraki.com/api/v1

Reviewer: ```suggestion
> - United States (FedRAMP): https://api.gov-meraki.com/api/v1
```
